### ISSUE TITLE
Time interval tests fail when in some timezone

### ISSUE THREAD

**Issue by taketime** [kkaefer/chrono.js#12]
If the system time is set to some timezones (examples: ALMT (Kazakhstan), CST (China)), time interval tests fail.

```

$ make test
   chrono.test.js test time intervals: AssertionError: ["1 year","1 day"] deepEqual ["1 year","2 days"]
    at Test.exports.test time intervals [as fn] (/Users/chris/mapbox/chrono.js/test/chrono.test.js:149:12)
    at Test.runParallel (/Users/chris/.nvm/v0.8.26/lib/node_modules/expresso/bin/expresso:959:10)
    at Test.run (/Users/chris/.nvm/v0.8.26/lib/node_modules/expresso/bin/expresso:924:18)
    at next (/Users/chris/.nvm/v0.8.26/lib/node_modules/expresso/bin/expresso:867:22)
    at Test.run (/Users/chris/.nvm/v0.8.26/lib/node_modules/expresso/bin/expresso:925:13)
    at next (/Users/chris/.nvm/v0.8.26/lib/node_modules/expresso/bin/expresso:867:22)
    at Test.run (/Users/chris/.nvm/v0.8.26/lib/node_modules/expresso/bin/expresso:925:13)
    at next (/Users/chris/.nvm/v0.8.26/lib/node_modules/expresso/bin/expresso:867:22)
    at Test.run (/Users/chris/.nvm/v0.8.26/lib/node_modules/expresso/bin/expresso:925:13)
    at next (/Users/chris/.nvm/v0.8.26/lib/node_modules/expresso/bin/expresso:867:22)


   Failures: 1



make: *** [test] Error 1
```

**Comment by taketime** [comment]
There failures happen only in timezones ahead of GMT.

**Comment by kkaefer** [comment]
This is fixed in a3a90592161fbad073c1aeec5d2209b13d14bb2e
